pkg/controller/controller_binding.go

```diff
@@ -396,17 +444,13 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 					return err
 				}
 
-				// TODO: We need to delete the ServiceInstanceCredential from the
-				// Broker since the Bind request was successful. This needs to be
-				// addressed as part of orphan mitigiation.
-
-				return nil
+				return c.setAndUpdateOrphanMitigation(binding, toUpdate, instance, serviceClass, brokerName, "too much time has elapsed")
 			}
 
 			if _, err := c.updateServiceInstanceCredentialStatus(toUpdate); err != nil {
```

Comment:
This call to updateServiceInstanceCredentialStatus should be removed. We should wait to update the status after setting up the binding for orphan mitigation.